boa/Cargo.toml

```diff
@@ -26,6 +26,7 @@ bitflags = "1.2.1"
 indexmap = "1.4.0"
 ryu-js = "0.2.0"
 chrono = "0.4"
+float-cmp = "0.8.0"
```

Comment:
This should not be a dependency, it should be a `dev-dependency`, in order to not add unused dependencies to our main build.